test/manual/authentication/pytest_crypto.py

```diff
@@ -11,7 +11,9 @@
 def test_hashing():
     with open("/vagrant/hiera/settings.yaml", 'r') as stream:
         try:
-            cryptopath = yaml.load(stream)['crypto']['path']
+            yamlres = yaml.load(stream)
+            root = yamlres['general']['root']
+            cryptopath = root + yamlres['crypto']['path']
         except yaml.YAMLError as exc:
             print exc
```

Comment:
Ah, you're right.  Your implementation is correct as is, since manual implementation of the unit test(s) require a terminal console, and will produce a standard output (including your exception) in the same console.  Can you make a trivial change of `except yaml.YAMLError as exc:` to `except yaml.YAMLError as error:`